<launch>
<!-- LDROBOT LiDAR message publisher node -->
 <node name="ldlidar_publisher_ld14p" pkg="ldlidar" type="ldlidar_net" output="screen">
  <param name="product_name" value="LDLiDAR_LD14P"/>
  <param name="topic_name" value="scan"/>
  <param name="frame_id" value="laser"/>

  <param name="device_ip" value="192.168.1.200"/>                 #雷达源IP
  <param name="device_port" value="2368"/>                        #雷达目的端口号
  <param name="difop_ip" value="192.168.1.102"/>                  #雷达目的ip
  <param name="difop_port" value="2369"/>                         #雷达源端口号

  <!--乐动LD-14雷达参数-->
  <!--angle1_start：屏蔽角度起始值 angle1_end：屏蔽角度结束值 
  角度顺时针方向
  注意：二者均不可为0，不开启角度屏蔽默认扫描角度为0°~360° -->
  <!-- <param name="flag_parted" type="bool" value="false"/> -->   <!--false:不屏蔽角度 true:屏蔽角度-->
  <!--Example: disable_min=[40,90]  disable_max=[50,120] 表示雷达扫描时屏蔽40°~50°,90°~120°的区间-->

  <!-- <rosparam param="disable_min">[120]</rosparam>  --><!--角度左值-->
  <!-- <rosparam param="disable_max">[240]</rosparam> --> <!--角度右值-->

 </node>

</launch>
